docs/UserGuide.md

```diff
@@ -384,56 +458,146 @@ Format: `c-clear`
+should be less or equal to 2.)<br>
 
-<div markdown="span" class="alert alert-primary">:bulb: 
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Notes about Employee Directory:**<br>
+
+* If _Employee Directory_ currently shows a list of archived employees (archived employees are displayed with a red
+ color label, named as `[Archived]`, in front of the name), command `c-active-list` needs to be used first to see a
+  list of all your employees. Then, `c-archive INDEX` can be used to archive a specified employee identified
+  by `INDEX`.<br>
 
-**Tip:**
-Command `c-list` can be used first to show a list of all active employees' contact details. 
-Then, `c-archive INDEX` can be used to archive a specific employee.
 </div>
 
-Examples:
-* `c-list` followed by `c-archive 2` archives the 2nd person in the employees' contact details.
-* `c-find Betsy` followed by `c-archive 1` archives the 1st person in the results of the `find` command.
+Example:
+* Let's say one of your employees, Alex Yeoh, is no longer working in your bubble tea shop. But he may come back
+to work again in the future. You, as a store manager, want to remove him from your employee directory without
+permanently deleting his information, so that you can retrieve/move back his information to your
+employee directory when he comes back to work again. In this case, you can follow these steps.
+
+Steps to archive the first employee, Alex Yeoh, in the sample data:
+1. Type `c-archive 1` into the _Command Box_.
+2. Press "Enter" to execute.
+
+
+Outcome:
+1. The _Result Display_ will show a success message with the employee's name that you have archived.
+2. Alex Yeoh's information will be removed from _Employee Directory_ pane.
+
+{Screenshot}
 
 #### 3.11 Archiving all employees : `c-archive-all`
 
-Archives all employees' contact details from the tCheck.
+Archives all employees in the _Employee Directory_ pane.
 
 Format: `c-archive-all`
 
+<div markdown="span" class="alert alert-primary">
+
+**:information_source: Notes about Employee Directory:**<br>
+
+* If _Employee Directory_ currently shows a list of archived employees (archived employees are displayed with a red
+ color label, named as `[Archived]`, in front of their names), command `c-active-list` needs to be used first to see a
+  list of all your employees. Then, `c-archive-all` can be used to archive all employees.<br>
+
+</div>
+
+Example:
+* Let's say after you familiarize yourself with tCheck, you want to remove all sample data in employee directory. But
+ you don't want to permanently delete those data, so that you can use them as a reference in the future. In this case
+ , you can follow these steps to archive all employees.
+
+Steps to archive all employees in the sample data:
+1. Type `c-archive-all` into the _Command Box_.
+2. Press "Enter" to execute.
+
+
+Outcome:
+1. The _Result Display_ will show a success message.
+2. All employees will be removed from _Employee Directory_ pane.
```

Comment:
Same issue here.

Reply:
It has been changed in the latest commit. Thanks!